setup.py

```diff
@@ -50,8 +50,12 @@
             'pytest>=2.7',
             'requests-mock',
             'xmltodict',
+            'shopify_python==0.1.2',
         ],
     },
+    dependency_links=[
+        'git+https://github.com/Shopify/shopify_python.git@v0.1.2#egg=shopify_python-0.1.2',
```

Comment:
`-e git` would be syntactically incorrect:
```
InvalidSchema: No connection adapters were found for '-e git+https://github.com/Shopify/shopify_python.git@v0.1.2'
```

These are strictly links, not installation instructions (e.g. `pip -e <dependency>` states that installation from version controlled directories should be via symlink and not file copying).